test/set.js

```diff
@@ -634,4 +634,22 @@ describe('Set', function () {
 
   it.skip('should throw proper errors when user invokes methods with wrong types of receiver', function () {
   });
+
+  it('SetIterator identification', function () {
```

Comment:
we should conditionally skip this first test when `Object.getPrototypeOf` is not available (in a separate `it`; same in the Map tests)